src/vector/awkward.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019-2020, Jim Pivarski, Eduardo Rodrigues, and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/vector for details.
+
+import awkward1 as ak
+import numpy as np
+
+from .common.lorentz import LorentzXYZCommon
+
+class LorentzXYZ(ak.Record, LorentzXYZCommon):
+    def __repr__(self):
+        return "Lxyz({0:.3g} {1:.3g} {2:.3g} {3:.3g})".format(self.x, self.y, self.z, self.t)
+
+class LorentzXYZArray(ak.Array, LorentzXYZCommon):
+    pass
+
+
+# This function only works as a ufunc overload, but it 
```

Comment:
Seems some words got deleted

Reply:
That was missing in Jim's original example as well.

Reply:
Not a reason not to improve, right? At least let's remove the now-spurious ", but it".